test-runner/src/main/scala/scala/scalanative/testinterface/ScalaNativeFramework.scala

```diff
@@ -35,11 +33,4 @@ class ScalaNativeFramework(val framework: Framework,
   }
 
   private[testinterface] def runDone(): Unit = _runner = null
-
-  private def fetchFrameworkInfo(): FrameworkInfo = {
-    _runner.send(Command.SendInfo(id, None))
-    val Command.SendInfo(_, Some(infos)) = _runner.receive()
-    infos
-  }
```

Comment:
Normally I wouldn't say that, but let's keep this, commented out, with a comment explaining why we're not using it at the moment, and that we should reenable it in the future.